### DISCUSSION TITLE
Problema no Gruntfile: BrowserSync + Jekyll

### DISCUSSION THREAD

**Discussion by pixelcriativo** [frontendbr/forum#622] [Perguntas e Respostas]
Boa tarde amigos.
Estou trancado num problema no Gruntfile de um projeto, exatamente o que houve para que ele tenha parado de funcionar eu não sei, mas acredito que seja algo relacionado ao watch do browserSync.

Se alguém puder me ajudar ou me cobrar algum valor para me ajudar nisso, agradeço.

Gruntfile:
https://codepen.io/pixelcriativo/pen/VWodeG

A tarefa "deploy" funciona.
Porém ao rodar a tarefa "default", ou seja, apenas **grunt** e salvar algum arquivo monitorado pelo watch, como por exemplo...
-  **markdown** ele "tranca" e fica: _[BS] Reloading Browsers..._ eterno.
- **.scss** ele "tranca" e fica: _[BS] File event [change] : public\themes\project\dist\css\theme.css_

Alguma sugestão, não sei se fui bem claro.
Abs.

**Comment by felipefialho** [comment]
Já tentou colocar um tempo de delay para o watch? O Grunt tem dessas.

**Comment by pixelcriativo** [comment]
Adicionei o debounceDelay nas opções do watch, mas continua a mesma coisa. Fiz alguma tentativas agora pela manhã me parece que o Jekyll que pode estar com problemas. Pois quando ele vai rodar shell "jekyllSync" que tem o comando "bundle exec jekyll build --incremental" que ele acaba congelando.

_Running "sass:docs" (sass) task
Running "sass:themeProjSite" (sass) task
Running "shell:jekyllSync" (shell) task
[BS] File event [change] : public\assets\dist\css\docs.css
[BS] File event [change] : public\themes\project\dist\css\theme.css_

**Comment by leonardoelias** [comment]
Qual seria a versão do seu node ?

**Comment by pixelcriativo** [comment]
@leonardoelias 
É a v8.1.4, porém já tentei com a v6.11.1 e não tive sucesso.
